inference-engine/thirdparty/clDNN/src/gpu/loop_gpu.cpp

```diff
@@ -125,7 +125,7 @@ struct loop_gpu : typed_primitive_impl<loop> {
         auto& iteration_mem = instance.input_iteration_mem;
         auto body_network = instance.get_body_network();
         const auto& input_primitive_map = node.get_input_mappings();
-        const int inputs_memory_count = instance.inputs_memory_count();
+        const int inputs_memory_count = static_cast<int>(instance.inputs_memory_count());
```

Comment:
I think we can set most stuffs with size_t

Reply:
Okay, I just fix to build issue but i think so. I will change it.

Reply:
Fixed it